rbi/core/array.rbi

```diff
@@ -711,6 +737,14 @@ class Array < Object
   sig {returns(T::Array[Elem])}
   def transpose(); end
 
+  sig do
+    params(
+        arrays: T::Array[T.untyped]
```

Comment:
WDYT about `T::Array[OtherElem]` in argument and `T::Array[T.any(Elem, OtherElem)]`?

Reply:
Interesting, yeah that sounds like a good idea to me. I need to declare OtherElem as a type alias alongside Elem at the top of the file, I assume?